cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -413,35 +456,93 @@ def parallel_single_qubit_randomized_benchmarking(
         num_circuits: The number of random circuits generated for each
             number of Cliffords.
         repetitions: The number of repetitions of each circuit.
+    Returns:
+        A dictionary from qubits to RandomizedBenchMarkResult objects.
+    """
+    return parallel_single_qubit_rb(
+        sampler,
+        qubits,
+        RBParameters(
+            num_clifford_range=num_clifford_range,
+            num_circuits=num_circuits,
+            repetitions=repetitions,
+            use_xy_basis=use_xy_basis,
+            strict_basis=False,
+        ),
+    )
+
 
+@overload
+def parallel_single_qubit_rb(
+    sampler: cirq.Sampler,
+    qubits: cirq.Qid,
+    parameters: RBParameters = RBParameters(),
+    rng_or_seed: np.random.Generator | int | None = None,
+) -> RandomizedBenchMarkResult: ...
+
+
+@overload
+def parallel_single_qubit_rb(
+    sampler: cirq.Sampler,
+    qubits: Sequence[cirq.Qid],
+    parameters: RBParameters = RBParameters(),
+    rng_or_seed: np.random.Generator | int | None = None,
+) -> ParallelRandomizedBenchmarkingResult: ...
+
+
+def parallel_single_qubit_rb(
+    sampler: cirq.Sampler,
+    qubits: Sequence[cirq.Qid] | cirq.Qid,
+    parameters: RBParameters = RBParameters(),
+    rng_or_seed: np.random.Generator | int | None = None,
+) -> RandomizedBenchMarkResult | ParallelRandomizedBenchmarkingResult:
+    """Clifford-based randomized benchmarking (RB) single qubits in parallel.
```

Comment:
Can we change this to always return `ParallelRandomizedBenchmarkingResult`?
This would lift ambiguity in the returned type - if users call parallel_single_qubit_rb they'd get ParallelRandomizedBenchmarkingResult.  Currently they'd have different result types for `qubits=q` and `qubits=[q]` although both pass one qubit.

Reply:
In that case, can you please add `single_qubit_rb` which would be a wrapper to 
```
parallel_single_qubit_rb(..., [q]).results_dictionary[q]
```
(a bonus solution would be to have a local function that returns result_dictionary and then convert it to either RandomizedBenchMarkResult or ParallelRandomizedBenchmarkingResult in single_qubit_rb and parallel_single_qubit_rb resp.)

BTW, the new deprecation messages both suggest to use `single_qubit_rb` which does not exist yet.  Let us have the deprecation of `single_qubit_randomized_benchmarking` suggest `single_qubit_rb` and `parallel_single_qubit_randomized_benchmarking` suggest `parallel_single_qubit_rb`.